zebra-chain/src/orchard/note.rs

```diff
@@ -26,9 +19,21 @@ mod nullifiers;
 pub use ciphertexts::{EncryptedNote, WrappedNoteKey};
 pub use nullifiers::Nullifier;
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Copy, Debug)]
+///
```

Comment:
Minor: Missing documentation?

Reply:
Minor: Not sure this is correct, but I think you meant `A uniformly sampled random seed`?

Reply:
It does sound weird... I based it on the spec which says "Choose uniformly random rseed" but I parsed that wrong (it would be better "A uniformly chosen random rseed" if I wanted to follow the spec, I guess).

After checking another similar occurrences in the code base I saw that they all use just "A random <value>" (e.g. "A random commitment trapdoor") so I went with something similar in c7af63050